docs/framework/wcf/feature-details/generating-a-wcf-client-from-service-metadata.md

```diff
@@ -12,7 +12,7 @@ This topic describes how to use the various switches in Svcutil.exe to generate
   
 - MEX request to the supplied address with `/mex` appended.  
   
-- DISCO request (using the [DiscoveryClientProtocol](https://go.microsoft.com/fwlink/?LinkId=94777) from ASP.NET Web services) to the supplied address.  
+- DISCO request (using the [DiscoveryClientProtocol](https://docs.microsoft.com/dotnet/api/system.web.services.discovery.discoveryclientprotocol&view=netframework-4.8) from ASP.NET Web services) to the supplied address.  
```

Comment:
this is producing a broken link even though the redirect link is still correct. I believe it's because the view information comes after the redirectedFrom info
https://docs.microsoft.com/en-us/dotnet/api/system.web.services.discovery.discoveryclientprotocol?redirectedfrom=MSDN&view=netframework-4.8. You could remove the view parameter as well.

But anyway, this should be the fix
```suggestion
- DISCO request (using the <xref:System.Web.Services.Discovery.DiscoveryClientProtocol> from ASP.NET Web services) to the supplied address.  
```